runtime/near-vm-runner/src/features.rs

```diff
@@ -10,7 +10,7 @@ mod opts {
     pub(super) const TAIL_CALL: bool = false;
     pub(super) const MULTI_MEMORY: bool = false;
     pub(super) const MEMORY64: bool = false;
-    pub(super) const SATURATING_FLOAT_TO_INT: bool = false;
+    pub(super) const SATURATING_FLOAT_TO_INT: bool = true;
```

Comment:
@nagisa, could you please verify if this is correct and this is how we want to make the change?

Reply:
Yeah this is okay.